packages/root-cms/ui/components/DocPickerModal/DocPickerModal.tsx

```diff
@@ -165,23 +333,70 @@ DocPickerModal.DocCard = (props: {
           <div className="DocPickerModal__DocCard__content__header__docId">
             {doc.id}
           </div>
+          {props.enableStatusBadges && <DocStatusBadges doc={doc} />}
         </div>
         <div className="DocPickerModal__DocCard__content__title">
           {previewTitle || '[UNTITLED]'}
         </div>
         <div className="DocPickerModal__DocCard__content__url">{liveUrl}</div>
       </div>
       <div className="DocPickerModal__DocCard__controls">
-        <Button
-          color="blue"
-          size="xs"
-          onClick={() => props.onDocSelected(props.doc)}
-        >
-          Select
-        </Button>
+        {props.showOpenInNewTab && (
+          <Button
+            variant="subtle"
+            color="gray"
+            size="xs"
+            onClick={() => {
+              const url = `/cms/content/${collection}/${slug}`;
+              window.open(url, '_blank');
+            }}
+            style={{padding: '0 8px'}}
+          >
+            <IconExternalLink size={16} />
+          </Button>
+        )}
+        {props.multiSelect ? (
+          selected ? (
+            <Button
+              variant="light"
+              color="blue"
+              size="xs"
+              onClick={() => {
+                setSelected(false);
+                props.onDocUnselected(props.doc);
+              }}
+            >
+              Unselect
+            </Button>
+          ) : (
+            <Button
+              variant="filled"
+              color="blue"
+              size="xs"
+              onClick={() => {
+                setSelected(true);
+                props.onDocSelected(props.doc);
+              }}
+            >
+              Select
+            </Button>
+          )
+        ) : (
+          <Button
+            color="blue"
+            size="xs"
+            onClick={() => props.onDocSelected(props.doc)}
+          >
+            Select
+          </Button>
+        )}
       </div>
     </div>
   );
 };
 
 DocPickerModal.id = MODAL_ID;
+
+// Backwards compatibility: export useDocSelectModal as an alias
+export const useDocSelectModal = useDocPickerModal;
```

Comment:
can we just rename these as appropriate? i don't think we need these aliases.

Reply:
done